### PR TITLE
Experimental group

### PR DESCRIPTION
**Pull request by tbayvas**
## Summary :chart_with_upwards_trend: ##
simplified group component card

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -16,6 +17,10 @@ export default function App() {
           name="Dashboard"
           component={DashboardStack}
         />
+        <Stack.Screen
+          name="View Group"
```
Comment (Peiyj) [inline App.js:21]: nit: the name should be Dashboard since it belongs to Dashboard component

File: components/CommonCompGroupCard.js [hunk 2]
```diff
@@ -1,61 +1,40 @@
-import React, { Component } from 'react';
```
Comment (Peiyj) [inline components/CommonCompGroupCard.js:1]: There seems to be a component exception about <Text> component in this file.

File: screens/Dashboard.js [hunk 3]
```diff
@@ -56,12 +57,10 @@ class Dashboard extends Component {
             <ScrollView style={{
                 alignSelf: 'stretch',
             }}>
-
-              {groups.map(group => <TouchableOpacity onPress={() => {this.viewGroup(group.groupID)}} key={groups.groupID}><CommonCompGroupCard groupName={group.groupName}
+              
+              {groups.map(group => <TouchableOpacity onPress={() => {this.viewGroup(group.groupID)}} value={groups.groupID}><CommonCompGroupCard 
                                               goal={group.goal}
```
Comment (Peiyj) [inline screens/Dashboard.js:62]: Are we removing these attributes from the card component?

File: screens/Dashboard.js [hunk 4]
```diff
@@ -56,12 +57,10 @@ class Dashboard extends Component {
             <ScrollView style={{
                 alignSelf: 'stretch',
             }}>
-
-              {groups.map(group => <TouchableOpacity onPress={() => {this.viewGroup(group.groupID)}} key={groups.groupID}><CommonCompGroupCard groupName={group.groupName}
+              
+              {groups.map(group => <TouchableOpacity onPress={() => {this.viewGroup(group.groupID)}} value={groups.groupID}><CommonCompGroupCard
```
Comment (Peiyj) [inline screens/Dashboard.js:61]: Are we still using CommonCompGroupCard or ExperimentalGroupCard?

**Review by Peiyj — COMMENTED** [review]
Almost there! there are some empty files that you might want to remove as well.

**Review by jakerogers2022 — CHANGES_REQUESTED** [review]
Causes "Error: Text strings must be rendered within a <Text> component." when run on mobile

**Review by jakerogers2022 — APPROVED** [review]

**Review by jakerogers2022 — APPROVED** [review]

**Pull request merged**